server/src/main/java/org/opensearch/common/blobstore/AsyncMultiStreamBlobContainer.java

```diff
@@ -14,12 +14,12 @@
 import java.io.IOException;
 
 /**
- * An extension of {@link BlobContainer} that adds {@link VerifyingMultiStreamBlobContainer#asyncBlobUpload} to allow
+ * An extension of {@link BlobContainer} that adds {@link AsyncMultiStreamBlobContainer#asyncBlobUpload} to allow
  * multipart uploads and performs integrity checks on transferred files
  *
  * @opensearch.internal
  */
-public interface VerifyingMultiStreamBlobContainer extends BlobContainer {
+public interface AsyncMultiStreamBlobContainer extends BlobContainer {
```

Comment:
The intent of `VerifyingMultiStreamBlobContainer` was an indication that integrity checks are implicit.